provider/azure/metadata_windows_test.go

```diff
@@ -0,0 +1,29 @@
+// +build windows
+
+package azure
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestLoadCustomData(t *testing.T) {
+
+	customDataPath = "testdata\\CustomData.bin"
```

Comment:
It's probably good hygiene to reset this to its original value with a `defer` block..